eng/Publishing.props

```diff
@@ -98,11 +98,16 @@
       <IsStableBuild>false</IsStableBuild>
       <IsStableBuild Condition="'$(DotNetFinalVersionKind)' == 'release'">true</IsStableBuild>
     </PropertyGroup>
-    
+
+    <!-- In VMR inner-build copy to local storage, do not push to AzDO -->
+    <PropertyGroup Condition="'$(DotNetBuildPhase)' == 'InnerRepo'">
+      <PushToLocalStorage>true</PushToLocalStorage>
+    </PropertyGroup>
```

Comment:
This will be moved into arcade's target, right?

Reply:
Yes, it's pending, on my machine - finishing testing of a batch of changes.